cli/install/install.go

```diff
@@ -0,0 +1,617 @@
+package install
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// FlagInstall is a struct that contains all install flags.
+type FlagInstall struct {
+	// InstallReinstall is a flag for reinstall option.
+	InstallReinstall bool
+	// InstallReinstall is a flag for force option.
+	InstallForce bool
+	// InstallReinstall is a flag for verbose option.
+	InstallVerbose bool
+	// InstallNoclean is a flag for no-clean-tmp option
+	InstallNoclean bool
+}
+
+// Packet is a struct containing sys and install name of the packet.
+type Packet struct {
+	// sysName is a string containing system name of packet.
+	sysName string
+	// installName is a string containing install name of packet.
+	installName string
+}
+
+// DistroInfo is a struct containing info about linux distro.
+type DistroInfo struct {
+	Name         string
+	Vendor       string
+	Version      string
+	Architecture string
+}
+
+var (
+	// PrettyName is a regexp for PrettyName in os-release file.
+	PrettyName = regexp.MustCompile(`^PRETTY_NAME=(.*)$`)
+	// ID is a regexp for ID in os-release file.
+	ID = regexp.MustCompile(`^ID=(.*)$`)
+	// VersionID is a regexp for VersionID in os-release file.
+	VersionID = regexp.MustCompile(`^VERSION_ID=(.*)$`)
+	// TarantoolLink is a link to the git repository of tarantool.
+	TarantoolLink = "https://github.com/tarantool/tarantool.git"
+	// TTLink is a link to the git repository of tt.
+	TTLink = "https://github.com/tarantool/tt"
+)
+
+// getArchitecture returns Architecture of machine.
+func getArchitecture() (string, error) {
```

Comment:
Same result, but shorter.

```
// GetArch returns Architecture of machine.                                                               
func GetArch() (string, error) {                                                                          
    out, err := exec.Command("uname", "-m").Output()                                                      
    if err != nil {                                                                                       
        return "", err                                                                                    
    }                                                                                                     
                                                                                                          
    return string(out), nil                                                                               
} 
```

Reply:
And move this function to util.go named as GetArch() please (It will be reused later)

Reply:
Changed and moved